README.md

```diff
@@ -102,6 +104,19 @@ pip install git+https://github.com/Storia-AI/sage.git@main
     export ANTHROPIC_API_KEY=...
     ```
 
+For easier configuration, create a `.sage-env` file with the following contents (change the API keys names based on your desired setup):
+```
+# Embeddings
+export OPENAI_API_KEY=
+# Vector store
+export PINECONE_API_KEY=
+# Reranking
+export NVIDIA_API_KEY=
+# Generation LLM
+export ANTHROPIC_API_KEY=
+# Github issues
+export GITHUB_TOKEN=
+```
```

Comment:
Good idea. I'd actually create this .sage-env file in the repo and just reference it here.